files/scripts/graphdb.sh

```diff
@@ -4,6 +4,13 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+function log {
+    local message="$1"
+    local timestamp
+    timestamp=$(date +"%Y-%m-%d %H:%M:%S")
```

Comment:
why not in the declaration above?

Reply:
It's a best practice when defining a `local`, according to my intellij plugin which cites https://github.com/koalaman/shellcheck/wiki/SC2155